src/core/annotation.js

```diff
@@ -343,6 +353,9 @@ class Annotation {
       modificationDate: this.modificationDate,
       rect: this.rectangle,
       subtype: params.subtype,
+      actions: collectActions(params.xref, dict, AnnotationActionEventType),
+      fieldName: this._constructFieldName(dict),
```

Comment:
Generally speaking, is it "correct" to always do this for *every* single Annotation since it probably doesn't make sense for a lot of them (e.g. LinkAnnotations. and probably more)?